Examples/Tests/reduced_diags/analysis_reduced_diags.py

```diff
@@ -61,28 +60,18 @@
 
 # PART2: get results from reduced diagnostics
 
-with open('./diags/reducedfiles/EF.txt') as csv_file:
-    csv_reader = csv.reader(csv_file, delimiter=',')
-    line_count = 0
-    for row in csv_reader:
-        if line_count == 2:
-            EF = np.array(row[2]).astype(np.float)
-        line_count += 1
-
-with open('./diags/reducedfiles/EP.txt') as csv_file:
-    csv_reader = csv.reader(csv_file, delimiter=',')
-    line_count = 0
-    for row in csv_reader:
-        if line_count == 2:
-            EP = np.array(row[2]).astype(np.float)
-        line_count += 1
+EFdata = np.genfromtxt("EF.txt")
+EPdata = np.genfromtxt("EP.txt")
+
+EF = EFdata[1][2]
+EP = EPdata[1][2]
```

Comment:
Why 1? Why 2?

Reply:
[0][] is the first data row, i.e. the row of step = 0.
[1][] is the second data row, i.e. the row of step = 200 in this case.
[1][2] is the second row and the third column which is the total energy.